RetailSuiteAccounting/UserInterface/01c_AccountSheet.md

```diff
@@ -0,0 +1,122 @@
+# Account sheet
+
+The *ACCOUNT SHEET* tab displays a chronologically ordered list of all movements in a specific account. The information displayed in the account sheet view is read-only.
+
+
+## Account sheet view without account selection
+
+*Accounting > Post > Tab ACCOUNT SHEET*
+
+![Account sheet - no account selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/AccountSheet/AccountSheetNoAccount.png "[Account sheet - no account selected]")
+
+When clicking the *ACCOUNT SHEET* tab, a blank tab and the notice "No account selected" may be displayed. An account must be selected to display the corresponding details. If a posting is selected in the *POSTINGS* tab, the details of the account specified in the *Contra Account* field are displayed in the *ACCOUNT SHEET* tab. If the *Contra Account* field is empty, the details of the account specified in the *Account* field are displayed.
+
+For detailed information about accessing an account sheet, see [Access the account sheet](../Operation/02_ReviewAccount.md#access-the-account-sheet).
+
+
+## Account sheet view with account selection
+
+*Accounting > Post > Tab ACCOUNT SHEET*
+
+![Account sheet](../../Assets/Screenshots/RetailSuiteAccounting/Book/AccountSheet/AccountSheet.png "[Account sheet]")
```

Comment:
```suggestion


When clicking the *ACCOUNT SHEET* tab, a blank tab and the notice "No account selected" may be displayed. An account must be selected to display the corresponding details. If a posting is selected in the *POSTINGS* tab, the details of the account specified in the *Contra Account* field are displayed in the *ACCOUNT SHEET* tab. If the *Contra Account* field is empty, the details of the account specified in the *Account* field are displayed.

For detailed information about accessing an account sheet, see [Access the account sheet](../Operation/02_ReviewAccount.md#access-the-account-sheet).


## Account sheet view with account selection

*Accounting > Post > Tab ACCOUNT SHEET*

![Account sheet](../../Assets/Screenshots/RetailSuiteAccounting/Book/AccountSheet/AccountSheet.png "[Account sheet]")

The *ACCOUNT SHEET* tab displays a chronologically ordered list of all movements in a specific account. All fields are read-only.    
An account and/or contra account must be selected to display the corresponding details. Otherwise, the notice *No account selected* is displayed.    
You can either select an account in the *POSTING* tab and switch to the *ACCOUNT SHEET* tab or directly enter an account and/or contra account number in the input line in the bottom. For detailed information about accessing an account sheet, see [Access the account sheet](../Operation/02_ReviewAccount.md#access-the-account-sheet).   
If a contra account number is specified in the *Contra account* field of the input line, the account sheet for the corresponding contra account is displayed. If no contra account number is entered, but an account number is specified in the *Account* field of the input line, the account sheet for the corresponding account is displayed. 
```